src/API/Site/Controllers/MerchantCenter/ShippingTimeController.php

```diff
@@ -265,13 +269,69 @@ protected function get_schema_properties(): array {
 			],
 			'time'         => [
 				'type'              => 'integer',
-				'description'       => __( 'The shipping time in days.', 'google-listings-and-ads' ),
+				'description'       => __( 'The minimum shipping time in days.', 'google-listings-and-ads' ),
+				'context'           => [ 'view', 'edit' ],
+				'validate_callback' => [ $this, 'validate_shipping_times' ],
+			],
+			'max_time'     => [
+				'type'              => 'integer',
+				'description'       => __( 'The maximum shipping time in days.', 'google-listings-and-ads' ),
 				'context'           => [ 'view', 'edit' ],
-				'validate_callback' => 'rest_validate_request_arg',
+				'validate_callback' => [ $this, 'validate_shipping_times' ],
 			],
 		];
 	}
 
+	/**
+	 * Get the args schema for the controller.
+	 *
+	 * @return array
+	 */
+	protected function get_args_schema(): array {
+		$schema                         = $this->get_schema_properties();
+		$schema['time']['required']     = true;
+		$schema['max_time']['required'] = true;
+		return $schema;
+	}
```

Comment:
Just curious. Why we cannot just add the 'required' in the array inside `get_schema_properties` method ?

Reply:
That was my initial approach, but the schema is shared between the `GET` and `POST` methods. If I make `time` and `max_time` required, the UI will also need to include those query parameters when fetching the times.